lisa/sut_orchestrator/azure/common.py

```diff
@@ -1696,6 +1698,10 @@ def get_or_create_storage_container(
     """
     Create a Azure Storage container if it does not exist.
     """
+    if "AZURE_STORAGE_ACCESS_TOKEN" in os.environ:
```

Comment:
minimized the access to the env variable. For here, the logic can be like below. the method is used to wrap the env check.
`credential = get_static_access_token() or credential`